helm_helpers_test.go

```diff
@@ -138,7 +138,7 @@ func Test_validateReleaseCharts(t *testing.T) {
 						Namespace:       "",
 						Enabled:         true,
 						Chart:           "stable/prometheus",
-						Version:         "",
+						Version:         "*",
```

Comment:
Why change `Version` from `""` to `"*"` ? Did this PR breaks this test ?

Reply:
I will update the "helm search" code to check for the empty string.